donor-portal/src/components/navbar/index.tsx

```diff
@@ -22,32 +22,33 @@ export default function NavBar() {
         <div className="logo" />
       </Link>
       <div className="nav-links">
-        <Link to="/about-us" className="nav-link">
+        <NavLink to="/" className="nav-link" exact>
+          Home
+        </NavLink>
+        <NavLink to="/about-us" className="nav-link">
           About
-        </Link>
-        <span>|</span>
-        <Link to="/donors" className="nav-link">
+        </NavLink>
+        <NavLink to="/donors" className="nav-link">
           Donors
-        </Link>
-        <span>|</span>
-        <Link to="/suppliers" className="nav-link">
-          Hospital and Suppliers
-        </Link>
-        <span>|</span>
+        </NavLink>
+        <NavLink to="/suppliers" className="nav-link">
+          Hospital & Suppliers
+        </NavLink>
         <Link to="/news-room" className="nav-link">
           Newsroom
         </Link>
         {userId != null ? (
-          <>
-            <span>|</span>
-            <a href="/" className="nav-link" onClick={() => handleLogout()}>
-              Logout
-            </a>
-          </>
+          <a href="/" className="nav-link" onClick={() => handleLogout()}>
+            Logout
+          </a>
         ) : (
-          <a href="/" className="login-btn" onClick={() => handleLogin()}>
+          <button
+            className="login-btn"
+            type="button"
```

Comment:
we probably don't need this?

Reply:
1) I used a button instead otherwise last child of type won't work
2) it only serves onClick

Reply:
Oh I mean the `type="buton"`. If we remove it, would the same CSS not work?

Reply:
Agree on using the `button` element here. It's more accurate.

Reply:
usually i never use type="buton" but ur linter throws an error then..